### ISSUE TITLE
Styles from root theme are not inherited

### ISSUE THREAD

**Issue by briosheje** [alexplumb/material-ui-phone-number#118]
Any style applied to material UI widgets defined on the theme of the application are not inherited by the component (including color, shapes and so on).

Might not be an issue at all, but it would at least be worth to mention it in the documentation

**Comment by gentlementlegen** [comment]
Agreed, global theming seems not to be working on the component. It is a bit tedious to duplicate the style just for this, would love a fix for it 😄
